src/App.css

```diff
@@ -11,8 +20,24 @@ html, body, #root, #root>div {
   font-weight: bold;
 }
 
-// Probably should make this sticky or something{}
+// Probably should make this sticky or something rather than flex{}
+.filterSidebar {
+  width: 350px;
+  float: left;
+  background: #27292b;
+  color: white;
+  padding-top: 20px;
+  padding-left: 200px;
+}
+
+li .ui.checkbox label, .ui.checkbox+label {
```

Comment:
This is a dangerously vague selector to have in a global CSS file. Use a more specific class.